bundles/org.openhab.binding.bambulab/README.md

```diff
@@ -161,20 +161,21 @@ CommandType:Parameter1:Parameter2:...
+| `AmsControl`         | `RESUME` / `RESET` / `PAUSE`                                                                                                             | Sends an AMS control command.                                                                     |
+| `PrintSpeed`         | `SILENT` / `STANDARD` / `SPORT` / `LUDICROUS`                                                                                            | Adjusts print speed.                                                                              |
+| `GCodeFile`          | `filename(string)`                                                                                                                       | Loads a G-code file.                                                                              |
+| `GCodeLine`          | `userId(string)\nlines(string...)`                                                                                                       | Sends multiple G-code lines. Lines are enter (`\n`) separated                                     |
+| `LedControl`         | (`CHAMBER_LIGHT` / `WORK_LIGHT`),(`ON` / `OFF` / `FLASHING`),`ledOnTime(int)?`,`ledOffTime(int)?`,`loopTimes(int)?`,`intervalTime(int)?` | Controls LED lighting.                                                                            |
+| `System`             | `GET_ACCESS_CODE`                                                                                                                        | Executes a system command.                                                                        |
+| `IpCamRecord`        | `enable(boolean)`                                                                                                                        | Starts or stops IP camera recording.                                                              |
+| `Info`               | `GET_VERSION`                                                                                                                            | Sends a info command.                                                                             |
+| `IpCamTimelaps`      | `enable(boolean)`                                                                                                                        | Enables or disables timelapse recording.                                                          |
+| `XCamControl`        | (`FIRST_LAYER_INSPECTOR` / `SPAGHETTI_DETECTOR`),`control(boolean)`,`printHalt(boolean)`                                                 | Controls XCam settings.                                                                           |
+| `Raw`                | `topic`,`raw-message`                                                                                                                    | Allows to send raw message to MQTT. Please add `%sequence_id%` in your message to put current ID. |
```

Comment:
We prefer not to expose raw channels and model everything to the openHAB domain. Is there some sepcific use for this?

Reply:
This is not OH's raw channel. This is an action that allows you to send raw message to the printer. Raw in this context belongs to JBambuAPI library

Reply:
Ah, part of the `sendCommand` Ok.